algorithms/src/Kokkos_Sort.hpp

```diff
@@ -584,13 +568,14 @@ struct min_max_functor {
 
 }  // namespace Impl
 
-template <class ExecutionSpace, class ViewType>
-std::enable_if_t<Kokkos::is_execution_space<ExecutionSpace>::value> sort(
-    const ExecutionSpace& exec, ViewType const& view,
-    bool const always_use_kokkos_sort = false) {
-  if (!always_use_kokkos_sort) {
-    if (Impl::try_std_sort(view, exec)) return;
-  }
+template <class ExecutionSpace, class DataType, class... Properties>
+std::enable_if_t<(Kokkos::is_execution_space<ExecutionSpace>::value) &&
+                 (not SpaceAccessibility<
+                     HostSpace, typename Kokkos::View<DataType, Properties...>::
+                                    memory_space>::accessible)>
```

Comment:
We don't use `not` anywhere else.
```suggestion
std::enable_if_t<(Kokkos::is_execution_space<ExecutionSpace>::value) &&
                 SpaceAccessibility<
                     HostSpace, typename Kokkos::View<DataType, Properties...>::
                                    memory_space>::accessible>
```

Reply:
Realistically, this branch is only executed now for GPU memory spaces and for those we want to call other libraries anyway (if they can be found).

Reply:
`not` changed into `!`.